### ISSUE TITLE
media-library:regenerate doesn't regenerate responsive images

### ISSUE THREAD

**Issue by riasvdv** [spatie/laravel-medialibrary#1949]
If you use

```php
$model
    ->addMediaFromRequest('image')
    ->withResponsiveImages()
```

And on a later date regenerate the images using `php artisan media-library:regenerate`, it won't regenerate the responsive images of the base image, only the responsive images of a conversion.

**Comment by artem-prozorov** [comment]
Hi! If deadline allows, I'd fix this within 1 month

**Comment by sebapastore** [comment]
Hi, I am trying to make my first contribution to open source. I have been looking into the code the past days and I want to ask your point of view before start coding.

I think that "media-library:regenerate" is working as espected. In the way I see, the command itself can't just know if you want or not responsive images to be generated for the base image. It could infer it by inspecting the "responsive_images" field,  but I think this wouldn't be an appropiete aproach.

I think it would be better to provide a bool parameter to the command, and based on this, generate or not responsive images for the base image.

**Comment by ctf0** [comment]
am also into this, if anyone would like to join forces, plz reply

**Comment by sebapastore** [comment]
Hi @ctf0 I would like to help un this one.

**Comment by sebapastore** [comment]
Perfect! 👌

**Comment by K03413R** [comment]
@ctf0 @sebapastore if you need another to help out. I'd love to join join forces with you two.

**Comment by ctf0** [comment]
@K03413R awesome, plz chk the PR, atm its only missing the tests, if u can help with that it would be perfect

**Comment by freekmurze** [comment]
This will be fixed by #2047
